### PR TITLE
Added podspec file

### PR DESCRIPTION
**Pull request by Recouse**
Added podspec file for #14 
I took version from SPM file. Also, you can add your email in `author` or remove it.

### CODE REVIEW

File: BricksAndTiles.podspec [hunk 1]
```diff
@@ -0,0 +1,24 @@
+Pod::Spec.new do |s|
+    s.name                  = 'BricksAndTiles'
+    s.version               = '0.2.0'
```
Comment (mugbug) [inline BricksAndTiles.podspec:3]: The correct current version would be `0.0.3`, right?
```suggestion
    s.version               = '0.0.3'
```

File: BricksAndTiles.podspec [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    s.summary               = 'A composable solution for building reusable lists in Swift/UIKit. As easy as playing with Legos.'
+  
+    s.homepage              = 'https://github.com/mugbug/BricksAndTiles'
+    s.license               = { :type => 'MIT', :file => 'LICENSE' }
+    s.author                = { 'mugbug' => '' }
```
Comment (mugbug) [inline BricksAndTiles.podspec:8]:
```suggestion
    s.author                = { 'mugbug' => 'zaronipedro@outlook.com' }
```

File: BricksAndTiles.podspec [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    s.source                = { :git => 'https://github.com/mugbug/BricksAndTiles.git', :tag => s.version.to_s }
+    s.swift_version         = '5.0'
```
Comment (mugbug) [inline BricksAndTiles.podspec:10]:
```suggestion
    s.swift_version         = '5.1'
```

File: BricksAndTiles.podspec [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    s.ios.deployment_target = '11.0'
+
+    s.dependency 'PaintAndBrush', '0.1.4'
```
Comment (mugbug) [inline BricksAndTiles.podspec:13]:
```suggestion
    s.dependency 'PaintAndBrush', '0.2.0'
```

**Review by mugbug — COMMENTED** [review]
Awesome! Thanks for your contribution. I just made some considerations about versions, could you give it a check, please?

**Comment by Recouse** [comment]
Updated the file

**Pull request merged**